scripts/lint/ourlint.js

```diff
@@ -201,6 +206,37 @@ const TESTS = {
       return `Invalid URLs:\n\n${invalidUrls.join('\n\n')}`;
     }
   },
+
+  /* Check if all licenses are valid SPDX identifiers */
+  async checkLicense(data) {
```

Comment:
As this function is the first async function in the ourlint script, you should also change the type of `TESTS` variable. The return of the functions must be `Promise<string | undefined> | string | undefined` instead of `string | undefined` now.

Note that the consumer `Promise.all` accepts either promises or not.